src/Resources/config/services.yml

```diff
@@ -1,15 +1,17 @@
 services:
-  shopsys.product_feed.google_bundle.feed_config:
-    class: Shopsys\ProductFeed\GoogleBundle\GoogleFeedConfig
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+
+  Shopsys\ProductFeed\GoogleBundle\GoogleFeedConfig:
     tags:
       - { name: shopsys.product_feed }
 
-  shopsys.product_feed.google_bundle.product_crud_extension:
-    class: Shopsys\ProductFeed\GoogleBundle\GoogleProductCrudExtension
+  Shopsys\ProductFeed\GoogleBundle\GoogleProductCrudExtension:
     tags:
       - { name: shopsys.crud_extension, type: product }
 
-  shopsys.product_feed.google_bundle.product_form_type:
-    class: Shopsys\ProductFeed\GoogleBundle\GoogleProductFormType
+  Shopsys\ProductFeed\GoogleBundle\GoogleProductFormType:
```

Comment:
this entry is unnecessary from now on :slightly_smiling_face:

Reply:
fixed